test/testUtils.ts

```diff
@@ -41,3 +41,9 @@ export const daysFromToday = (days: number): Date => {
   nyDato.setDate(nyDato.getDate() + days);
   return new Date(nyDato);
 };
+
+export const daysFromDate = (date: Date, days: number): Date => {
```

Comment:
I stedet for å lage en egen metode her, kan man kanskje bruke dayjs? `date.add(days, "days")`